### DISCUSSION TITLE
Behavior of monitorEventLoopDelay when called from a Worker

### DISCUSSION THREAD

**Discussion by jeluard** [nodejs/help#4660] [Q&A]
When called from a [Worker](https://nodejs.org/api/worker_threads.html), what is the expected behavior from [monitorEventLoopDelay](https://nodejs.org/api/perf_hooks.html#perf_hooksmonitoreventloopdelayoptions)?
Shall it report main JS thread event loop delay, or this worker thread's one? 

[eventLoopUtilization](https://nodejs.org/api/perf_hooks.html#performanceeventlooputilizationutilization1-utilization2) also exists on [Worker](https://nodejs.org/api/worker_threads.html#performanceeventlooputilizationutilization1-utilization2), suggesting `monitorEventLoopDelay` always monitors main JS thread?

**Comment by DarkGL** [comment]
It should report worker thread event loop delay

For example

```
const { Worker, isMainThread, parentPort } = require('worker_threads');
const { monitorEventLoopDelay } = require('perf_hooks');

if (isMainThread) {
  const worker = new Worker(__filename);
  worker.on('message', (message) => {
    console.log(`Received message from worker: ${message}`);
  });
} else {
  const histogram = monitorEventLoopDelay();
  histogram.enable();
  
  setTimeout(() => {
    histogram.disable();
    parentPort.postMessage(`Worker event loop delay: ${histogram.percentile(50)} ms`);
  }, 1000);
}
```
